classes/class-uabb-admin-settings-multisite.php

```diff
@@ -13,24 +19,22 @@ final class UABBBuilderMultisiteSettings {
 	 * @since 1.0
 	 * @return void
 	 */
-	static public function init()
-	{
+	static public function init() {
 		add_action( 'admin_init', __CLASS__ . '::admin_init' );
 		add_action( 'admin_init', __CLASS__ . '::uabb_lite_redirect_on_activation' );
-		add_action( 'network_admin_menu',                __CLASS__ . '::menu' );
+		add_action( 'network_admin_menu', __CLASS__ . '::menu' );
 	}
 
 	/**
-	 * Redirects to UABB Lite Welcome page on activation
+	 * Sets the activate redirect url to the network admin settings.
 	 *
-	 * @since 1.0
-	 * @return null
+	 * @since 1.8
```

Comment:
This function is available from version 1.0 and not 1.8. We are still on version 1.2.4

Reply:
Done